source/tyr/tyr/tasks.py

```diff
@@ -115,8 +124,26 @@ def import_data(
         'shape': shape2ed,
     }
 
+    new_job = None
     for _file in files:
-        filename = None
+        # we check if there's a job_id in file's name, there's no job id is found, we have to create a new_job
+        job_id = retrive_job_id(_file)
+        if not job_id:
+            new_job = models.Job()
+            new_job.instance = instance
+            new_job.state = 'running'
```

Comment:
Here we loose all the new_job created from second to second last  file without job id.